Firmware/Marlin_main.cpp

```diff
@@ -5370,60 +5370,38 @@ if(eSoundMode!=e_SOUND_MODE_SILENT)
 		}
 		#endif // SUPPORT_VERBOSITY
 
-		for (uint8_t i = 0; i < 4; ++i) {
-			unsigned char codes[4] = { 'L', 'R', 'F', 'B' };
-			long correction = 0;
+    const constexpr int8_t sides = 4;
+    long correction[sides] = {0, 0, 0, 0};
+    for (uint8_t i = 0; i < sides; ++i){
+      unsigned char codes[sides] = { 'L', 'R', 'F', 'B' };
```

Comment:
we can even make this array PROGMEM (saves on copying the array into RAM and RAM itself)